druid/src/widget/flex.rs

```diff
@@ -942,6 +943,113 @@ impl<T: Data> Widget<T> for Flex<T> {
             ..Default::default()
         }
     }
+
+    fn compute_max_intrinsic(
+        &mut self,
+        axis: Axis,
+        ctx: &mut LayoutCtx,
+        bc: &BoxConstraints,
+        data: &T,
+        env: &Env,
+    ) -> f64 {
+        match (self.direction, axis) {
+            (Axis::Vertical, Axis::Horizontal) | (Axis::Horizontal, Axis::Vertical) => {
```

Comment:
Maybe `if self.direction == axis` would be clearer?

Reply:
1000%